editors/emacs/lambdapi.el

```diff
@@ -14,10 +14,18 @@
 ;; Keywords
 (defconst lambdapi-font-lock-keywords
   (list (cons (concat "\\<" (regexp-opt '(
-    "require" "open" "as" "let" "in" "symbol" "const" "injective" "rule" "and"
-    "definition" "theorem" "assert" "assertnot" "set" "proof" "qed" "admit"
-    "abort" "focus" "print" "proofterm" "refine" "apply" "simpl" "assume"
-    "rewrite" "refl" "sym" "type" "compute"
+    "require" "open" "as" "let" "in"
+    "symbol" "constant" "injective"
+    "private protected"
+    "rule" "and" "definition" "theorem"
+    "assert" "assertnot" "set"
```

Comment:
Are you sure about "set"?

Reply:
methodology : take every keyword inside src/parserl.ml (after ``KW.create``) ... no more no less ... is this correct ?

Reply:
Right. I didn't remember the "set" command...